share/revng/test/tests/scripting/scripting.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env python3
+#
+# This file is distributed under the MIT License. See LICENSE.md for details.
+#
+
+import argparse
+import re
+
+from revng import ptml
+from revng.model import MetaAddress, MetaAddressType
+from revng.profile import SimpleCLI, SimpleDaemon
+
+parser = argparse.ArgumentParser()
```

Comment:
Split the code into two functions: `main` and `run_test`.
`main` should handle the argument parsing and creating the project object
`run_test` should have the remaining code
also add
```
if __name__ == "__main__":
  main()
```

Reply:
The `main` isn't creating the profile as said. `run_test` should look like:
```
def run_test(project_getter: Callable[[], Project], binary: str):
```